site/docs/Tutorial-GraphQLClients.md

```diff
@@ -0,0 +1,34 @@
+---
+title: GraphQL Clients
+layout: ../_core/DocsLayout
+category: Tutorial
+permalink: /graphql-js/graphql-clients/
+next: /graphql-js/basic-types/
+---
+
+Since a GraphQL API has more underlying structure than a REST API, there are more powerful clients like [Relay](https://facebook.github.io/relay/) which can automatically handling batching, caching, and other features. But you don't need a complex client to call a GraphQL server. With `express-graphql`, you can just send an HTTP POST request to the endpoint you mounted your GraphQL server on, passing the GraphQL query as the `query` field in a JSON payload.
```

Comment:
Can we add a curl example here? Also, I would mention that queries can be executed with an HTTP GET, but mutations must be executed with an HTTP POST.